test/integration/cypress/integration/NodePage.spec.ts

```diff
@@ -11,10 +11,27 @@ context("Node page", () => {
 
   it("should show edges by predicate", () => {
     // TODO: use fixture to get this
-    // page.edgeList("/r/IsA").edges.contains("Test node 414");
+    page.gridEdgeList("\\/r\\/AtLocation").list.should('exist')
   });
 
   it("should show the node datasource", () => {
     page.datasource.should("have.text", "gui_test_data");
   });
+
+  it("should have the grid tab selected by default", () => {
+    page.assertTabSelected(NodePageTab.PredicateGrid)
+  })
+
+  it("should route to the predicate list when the tab is clicked", () => {
+    page.getTab(NodePageTab.PredicateList).click()
+    cy.location().should((loc) => {
+      expect(loc.pathname).to.eq(`${page.relativeUrl}/list`)
+    })
+  })
+
+  it("should contain predicate lists in the list tab", () => {
+    page.visit("/list")
```

Comment:
I'd prefer URL construction details be in the page object and not spread between the page object and the spec.
visitList()?

Reply:
I knew you would complain about this.  But I added it anyways...